itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/TestAcidOnTez.java

```diff
@@ -681,11 +681,11 @@ public void testAcidInsertWithRemoveUnion() throws Exception {
     }
 
     String[][] expected2 = {
-      {"{\"writeid\":1,\"bucketid\":536870913,\"rowid\":0}\t1\t2", "warehouse/t/delta_0000001_0000001_0001/bucket_00000_0"},
-      {"{\"writeid\":1,\"bucketid\":536870913,\"rowid\":1}\t3\t4", "warehouse/t/delta_0000001_0000001_0001/bucket_00000_0"},
-      {"{\"writeid\":1,\"bucketid\":536870914,\"rowid\":0}\t5\t6", "warehouse/t/delta_0000001_0000001_0002/bucket_00000_0"},
-      {"{\"writeid\":1,\"bucketid\":536870914,\"rowid\":1}\t7\t8", "warehouse/t/delta_0000001_0000001_0002/bucket_00000_0"},
-      {"{\"writeid\":1,\"bucketid\":536870915,\"rowid\":0}\t9\t10", "warehouse/t/delta_0000001_0000001_0003/bucket_00000_0"}
+       {"{\"writeid\":2,\"bucketid\":536870913,\"rowid\":0}\t1\t2", "warehouse/t/delta_0000002_0000002_0001/bucket_00000"},
+       {"{\"writeid\":2,\"bucketid\":536870913,\"rowid\":1}\t3\t4", "warehouse/t/delta_0000002_0000002_0001/bucket_00000"},
+       {"{\"writeid\":2,\"bucketid\":536870914,\"rowid\":0}\t5\t6", "warehouse/t/delta_0000002_0000002_0002/bucket_00000"},
+       {"{\"writeid\":2,\"bucketid\":536870914,\"rowid\":1}\t7\t8", "warehouse/t/delta_0000002_0000002_0002/bucket_00000"},
+       {"{\"writeid\":2,\"bucketid\":536870915,\"rowid\":0}\t9\t10", "warehouse/t/delta_0000002_0000002_0003/bucket_00000"}
```

Comment:
Could you please modify the previous comment to match the code?

Reply:
Sorry, did not get understand your comment.

Reply:
In the code the previous comment contains the expected directory/file structure. We should fix that as well